docker-image/v1.8/debian-elasticsearch7/Gemfile.lock

```diff
@@ -124,11 +126,12 @@ PLATFORMS
 
 DEPENDENCIES
   elasticsearch (~> 7.0)
-  elasticsearch-xpack (~> 7.4.0)
+  elasticsearch-xpack (~> 7.0)
   ffi
   fluent-plugin-concat (~> 2.4.0)
+  fluent-plugin-dedot_filter (~> 1.0)
   fluent-plugin-detect-exceptions (~> 0.0.12)
-  fluent-plugin-elasticsearch (~> 3.7.1)
+  fluent-plugin-elasticsearch (~> 4.0)
```

Comment:
Contaminated updating Elasticsearch plugin.
We shouldn't touch plugins and dependent gems versions.

Reply:
No problem, I revert it and I will push a new PR :)